src/invariant/InvariantUtils.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "xdr/Stellar-ledger-entries.h"
+
+namespace stellar
+{
+bool
+trustLineFlagIsValid(uint32_t flag)
+{
+    return flag <= AUTHORIZED_FLAG;
```

Comment:
This should probably use bitwise comparisons rather than inequality comparisons, to reflect the fact that this is a flags field.